src/Servers/IIS/IIS/src/Core/WrappingStream.cs

```diff
@@ -107,7 +107,6 @@ public override int EndRead(IAsyncResult asyncResult)
         public override void EndWrite(IAsyncResult asyncResult)
             => _inner.EndWrite(asyncResult);
 
-        [Obsolete("This Remoting API is not supported and throws PlatformNotSupportedException.", DiagnosticId = "SYSLIB0010", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
This, and the other Obsolete attribute I removed for InitializeLifetimeService() were on internal types.